app/src/main/java/com/team/uptech/pomodoro/data/repository/impl/PomodoroRepositoryPrefsImpl.kt

```diff
@@ -0,0 +1,91 @@
+package com.team.uptech.pomodoro.data.repository.impl
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.preference.PreferenceManager
+import android.util.Log
+import com.team.uptech.pomodoro.data.model.Pomodoro
+import com.team.uptech.pomodoro.data.repository.PomodoroRepository
+import io.reactivex.Completable
+import io.reactivex.Single
+import io.reactivex.schedulers.Schedulers
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+
+/**
+ * Created on 28.04.17.
+ */
+class PomodoroRepositoryPrefsImpl @Inject constructor(context: Context) : PomodoroRepository {
```

Comment:
why do you need an @Inject constructor here? the point is to always provide an interface as you did in the module